### ISSUE TITLE
Incorrect parsing if ';' in value

### ISSUE THREAD

**Issue by ManushB** [peakwinter/python-nginx#30]
Version: **1.5.1**

Nginx Config Key/Value:
`add_header X-XSS-Protection "1;mode-block";`

Parse Output:
`{'conf': [{'add_header': 'X-XSS-Protection "1'}, {'mode-block"': ''}]}`

**Comment by peakwinter** [comment]
Fixed in #31 and version 1.5.2

**Comment by unknown321** [comment]
Version: **1.5.4**

Config:
```
if ( $http_user_agent = "Mozilla/4.0 (compatible; MSIE 6.0; Windows NT 5.1; SV1)" ) {
    return 403;
}
```

Stacktrace:

```
  File "test.py", line 2, in <module>
    nginx.loadf('nginx.conf')
  File "/home/user/.local/lib/python2.7/site-packages/nginx.py", line 570, in loadf
    return load(f)
  File "/home/user/.local/lib/python2.7/site-packages/nginx.py", line 560, in load
    return loads(fobj.read())
  File "/home/user/.local/lib/python2.7/site-packages/nginx.py", line 509, in loads
    if isinstance(lopen[0], Container):
IndexError: list index out of range
```

**Comment by peakwinter** [comment]
@unknown321 This should be fixed in 1.5.5. Thanks for reporting!
